wandb/data_types.py

```diff
@@ -850,9 +850,20 @@ def __init__(self, parts_path):
         self.parts_path = parts_path
         self._loaded_part_entries = {}
 
-    def to_json(self, artifact):
-        json_obj = super(PartitionedTable, self).to_json(artifact)
-        json_obj["parts_path"] = self.parts_path
+    def to_json(self, artifact_or_run):
+        json_obj = {
+            "_type": PartitionedTable._log_type,
+        }
+        wandb_run, _ = _safe_sdk_import()
```

Comment:
I assume there are other reasons for this, but we already [import the entire sdk](https://github.com/wandb/client/blob/master/wandb/__init__.py#L36-L40) in the core wandb module.   We should be able to just use `wandb.wandb_sdk.wandb_run` to get the appropriate module.  We're also importing other sdk classes conditionally at the top of this file... perhaps I'm opening a can of worms I shouldn't be.

Reply:
Well, at one point this was a fix for some circular dependencies. However, your comment coaxed me into just replacing these safe_sdk_import functions with their alternatives and everything seems to work. Going to run regression now.

Reply:
yeah, regressions passed!, so i stripped it out

Reply:
😎